### PR TITLE
Add slither module.

### PR DESCRIPTION
**Pull request by applmak**

### CODE REVIEW

File: demo_modules/slither/slither.js [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+  '#d02090',
+  '#8a2be2',
+]
+const SPEED = 0.2;
+
+const isOutOfBounds = ({x, y}, rect) => {
+  return x < rect.x || y < rect.y || x > rect.x + rect.w || y > rect.y + rect.h;
+};
+
+const darken = (color, i) => {
```
Comment (plural) [inline demo_modules/slither/slither.js]: is it worth using a real color object and only doing the conversion to strings for color fewer times ? I assume this runs fine as-is, so maybe just a TODO would be nice.

File: demo_modules/slither/slither.js [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+      position: {
+        x: 'NumberLerpInterpolator',
+        y: 'NumberLerpInterpolator'
+      },
+      color: 'ValueNearestInterpolator'
+    }]);
+    return Promise.resolve();
+  }
+  tick(time, delta) {
+    if (!time) {
```
Comment (plural) [inline demo_modules/slither/slither.js]: I filed #201 to track fixing the time checks.

File: demo_modules/slither/slither.js (hunk 1 above)
Comment (applmak) [reply demo_modules/slither/slither.js]: Done.

File: demo_modules/slither/slither.js (hunk 2 above)
Comment (applmak) [reply demo_modules/slither/slither.js]: Thanks!

**Pull request merged**